--- /home/msasongko17/research/reusetracker-test/testing-reusetracker-repo/ReuseTracker/hpctoolkit-externals/libunwind/libunwind/src/x86_64/Ginit.c	2022-10-09 16:28:56.736184669 +0300
+++ /home/msasongko17/research/reusetracker-test/testing-reusetracker-repo/ReuseTracker-working/hpctoolkit-externals/libunwind/libunwind/src/x86_64/Ginit.c	2022-10-09 12:41:00.958763552 +0300
@@ -49,7 +49,7 @@
 
 unw_addr_space_t unw_local_addr_space = &local_addr_space;
 
-HIDDEN unw_dyn_info_list_t _U_dyn_info_list;
+//HIDDEN unw_dyn_info_list_t _U_dyn_info_list;
 
 /* XXX fix me: there is currently no way to locate the dyn-info list
        by a remote unwinder.  On ia64, this is done via a special
